pkgs/development/libraries/gtk/patches/3.0-hardcode-gsettings.patch

```diff
@@ -0,0 +1,370 @@
+diff --git a/demos/gtk-demo/application.c b/demos/gtk-demo/application.c
+index 289f73a..9ce0466 100644
+--- a/demos/gtk-demo/application.c
++++ b/demos/gtk-demo/application.c
+@@ -384,7 +384,16 @@ demo_application_init (DemoApplication *app)
+   GSettings *settings;
+   GAction *action;
+ 
+-  settings = g_settings_new ("org.gtk.Demo");
++  {
++    g_autoptr(GSettingsSchemaSource) schema_source;
++    g_autoptr(GSettingsSchema) schema;
++    schema_source = g_settings_schema_source_new_from_directory("@gtk@",
++                                                                g_settings_schema_source_get_default(),
```

Comment:
This is actually pointless since we are calling `lookup` with `recursive=FALSE`.